src/CMakeLists.txt

```diff
@@ -64,6 +64,7 @@ set(GAMEENGINE_SRC
     game/client/view.cpp
     game/client/water.cpp
     game/client/gui/gamefont.cpp
+	game/client/image.cpp
```

Comment:
Tab rather than spaces here? Clang format doesn't handle the cmake I don't believe.